scripts/growgreat_extract/msisdn_from_details.py

```diff
@@ -4,7 +4,7 @@
 
 def get_msisdn(details):
     last = None
-    for addr, addr_dets in details.get("addresses", {}).get("msisdn", {}).items():
+    for addr, _addr_dets in details.get("addresses", {}).get("msisdn", {}).items():
```

Comment:
If we're not using the value, do we need to iterate over `.items()`?

Reply:
I have updated it as follows, and the tests still pass.  Is this what you meant?

`for addr in details.get("addresses", {}).get("msisdn", {}):
        if details.get("default") is True:
            return addr
        if details.get("optedout") is not True:
            last = addr
    return last`

Reply:
Yup, thanks. 🙂